networking/networking.go

```diff
@@ -116,6 +125,54 @@ func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, netList common
 	return &n, nil
 }
 
+// enableDefaultLocalnetRouting enables the route_localnet attribute on the supposedly default network interface.
+// This allows to setup a loopback NAT to access port forwardings to Pods on the host with the localhost address.
+func (n *Networking) enableDefaultLocalnetRouting() error {
+	routeLocalnetFormat := ""
+
+	defaultHostIP, err := n.GetDefaultHostIP()
+	if err != nil {
+		return err
+	}
+
+	defaultHostIPstring := defaultHostIP.String()
+	switch {
+	case strings.Contains(defaultHostIPstring, "."):
+		routeLocalnetFormat = "/proc/sys/net/ipv4/conf/%s/route_localnet"
+	case strings.Contains(defaultHostIPstring, ":"):
+		return fmt.Errorf("unexpected IPv6 Address returned for default host interface: %q", defaultHostIPstring)
+	default:
+		return fmt.Errorf("unknown type for default Host IP: %q", defaultHostIPstring)
+	}
+
+	hostIfaces, err := n.GetIfacesByIP(defaultHostIP)
+	if err != nil {
+		return err
+	}
+
+	for _, hostIface := range hostIfaces {
+
+		routeLocalnetPath := fmt.Sprintf(routeLocalnetFormat, hostIface.Name)
+		routeLocalnetValue, err := ioutil.ReadFile(routeLocalnetPath)
+		if err != nil {
+			return err
+		}
+		if string(routeLocalnetValue) != "1" {
```

Comment:
Going down to http://lxr.free-electrons.com/source/kernel/sysctl.c#L2066 it will make a more significant difference if `write != 0`. This might not even be worth discussing but it's also not worth changing IMHO.